desktop-widgets/divelistview.cpp

```diff
@@ -306,73 +306,6 @@ QList<dive_trip_t *> DiveListView::selectedTrips()
 	return ret;
 }
 
-void DiveListView::selectDive(int i)
```

Comment:
this code goes back to after the gtk -> qt port and the addition of the marble map widget.

Reply:
is this and the plural not uses elsewhere outside of map logic?